pkg/schedule/schedulers/evict_slow_store.go

```diff
@@ -308,7 +344,122 @@ func (s *evictSlowStoreScheduler) IsScheduleAllowed(cluster sche.SchedulerCluste
 // Schedule implements the Scheduler interface.
 func (s *evictSlowStoreScheduler) Schedule(cluster sche.SchedulerCluster, _ bool) ([]*operator.Operator, []plan.Plan) {
 	evictSlowStoreCounter.Inc()
+	s.scheduleNetworkSlowStore(cluster)
```

Comment:
Do we need to add a switch to disable it?

Reply:
When `MaxNetworkSlowStore` is 0, it is disable?